README.md

```diff
@@ -438,8 +439,8 @@ metric's computation function at a regular interval, and also has provisions
 for sending your metric other metrics, so that you can chain metrics
 together.
 
-The `ticker` is where you should return Messages (Gauge or Derived values) and
-keep track of how 'far long' you've computed, as to avoid returning the same
+The `ticker` is where you return Messages (Gauge or Derived values) to
+keep track of how 'far along' you've reached, in order to avoid returning the same
```

Comment:
Perhaps how 'far along' should be progress (I'd have gone for that if I actually understood from that text what it does).

Perhaps "to avoid returning the same message multiple times" should be "to signify that the newest instances of a message should be considered to supersede and/or logically be coalesced with previous occurrences of that message"

As I say, right now it doesnt convey to me the nature of the mechanism